src/Test/Utilities/Shared/Pe/BrokenStream.cs

```diff
@@ -68,7 +68,13 @@ public override void SetLength(long value)
         public override void Write(byte[] buffer, int offset, int count)
         {
             if (BreakHow == 0)
+            {
                 throw new IOException();
+            }
+            if (BreakHow == 3)
```

Comment:
I'll incorporate this when I solve your merge conflicts when flowing this into master.  (I don't want to have to run the test suite again right now...)